### DISCUSSION TITLE
power consumption?

### DISCUSSION THREAD

**Discussion by Cronvs** [ajfriesen/PokyPow#6] [Q&A] [answered]
Are there any rough numbers on the power consumption of the pokypow (even preliminary)?

**Comment by ajfriesen** [comment]
Good question.

The heart of this device is the ESP32-C3-MINI-1U.
That takes 3.0-3.6V, but since we are powering it over USB the USB2.0 and USB-C it is 5V.
I am not sure about the loss in between. 

The power consumption can be checked here:
- https://www.espressif.com/sites/default/files/documentation/esp32-c3-mini-1_datasheet_en.pdf
- Chapter 6 Electrical Characteristics and
- Chapter 6.4 Current Consumption Characteristics (**peak 350mA**)

And here:
- https://www.espressif.com/sites/default/files/documentation/esp32-c3_datasheet_en.pdf
- Chapter 5 Electrical Characteristics
- Chapter 5.6 Current Consumption Depends on the Power mode

I do not know the average value.

Why do you ask?

**Comment by Cronvs** [comment]
I have been looking for this kind of solution for some time now, of course at those wattage it's becoming slightly trivial (but I do enjoy efficiency), and nanoKVM is another project I have been eyeing, which claims to have 1 watt. But it's very overkill, gives too much control (security) and wastes hdmi data. I will probably invest in a few pokypow and would be cherry on top (for me) knowing this beats nanokvm in power.

**Reply by ajfriesen** [reply, marked as answer]
When the new revision is out I will be doing some measurements.

**Reply by Cronvs** [reply]
Super, thank you very much!

**Reply by lukasMega** [reply]
any news about power measurements?

**Reply by ajfriesen** [reply]
I just measured with a multimeter I have laying around, so nothing fancy.

With LED off:
- average 51mA
- peaking occasionally to 68mA

With LED set to white 100%:
- average 85mA
- peaking occasionally to 100mA

Edit:
Power via USB 2.0 from the mainboard which is 5V.

**Reply by Cronvs** [reply]
So, 0.25W to 0.5W, assuming 5V usb. Thank you.
